mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -200,14 +200,15 @@ def copy_file(self, source, destination):
 
         self.logger.debug("SHA1: %s", hashlib.sha1(aux_source).hexdigest())
 
-        if platform.system() == 'Windows':
-            self.logger.debug('copying file: "{}" to "{}"'.format(source, destination))
-            os.system("copy \"%s\" \"%s\"" % (os.path.abspath(source), destination))
-        else:
-            self.logger.debug("writing binary: %s (size=%i bytes)", destination, len(aux_source))
+        self.logger.debug("writing binary: %s (size=%i bytes)", destination, len(aux_source))
+        try:
             new_file = self.get_file(destination)
```

Comment:
this function need to tune a bit, but otherwise changes here already makes senses..